packages/vue-cli-plugin-vuetify/generator/templates/v3/src/vite.config.js

```diff
@@ -1,17 +1,22 @@
-import { defineConfig } from 'vite'
-import vue from '@vitejs/plugin-vue'
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+import vuetify from "@vuetify/vite-plugin";
 
-import path from 'path'
-const srcPath = path.resolve(__dirname, 'src', 'styles', 'variables.scss')
+import path from "path";
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [vue()],
-  define: { 'process.env': {} },
-  css: {
-    preprocessorOptions: {
-      sass: { additionalData: `@import ${srcPath}\n` },
-      scss: { additionalData: `@import ${srcPath};\n` },
+  plugins: [
+    vue(),
+    vuetify({
+      autoimport: true,
+      styles: "expose",
```

Comment:
`styles: "expose"` should only be set if you need to customise sass variables, and if it is then the `vuetify/styles` import should be moved to a sass file.